plugin/src/com/microsoft/alm/plugin/operations/AccountLookupOperation.java

```diff
@@ -105,4 +117,40 @@ protected void terminate(Throwable throwable) {
         onLookupResults(results);
         onLookupCompleted();
     }
+
+    // Make a server call to get the uris for the accounts the user has access to
+    private List<String> getAccountUris(final ServerContext vsoDeploymentContext) {
+        // new list of account uris to return
+        List<String> accountUris = new ArrayList<String>();
+
+        // Issue account request
+        final Client accountClient = vsoDeploymentContext.getClient();
+        final String accountApiUrlFormat = VsoAuthenticationProvider.VSO_AUTH_URL + ACCOUNT_ENDPOINT;
+        final WebTarget resourceTarget = accountClient.target(String.format(accountApiUrlFormat, vsoDeploymentContext.getUserId()));
+        final Map<String, String> parameters = new HashMap<String, String>();
+        parameters.put("api-version", "3.0-preview.1");
+        parameters.put("charset", "UTF-8");
+        final Invocation invocation = resourceTarget.request(
+                new MediaType("application", "json", parameters))
+                .buildGet();
+        final String response = invocation.invoke(String.class);
+
+        // Parse result tree
+        final ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode rootNode = null;
+        try {
+            rootNode = objectMapper.readTree(response);
+        } catch (IOException e) {
+            logger.error("Could not parse Account response", e);
```

Comment:
It seems more appropriate here to throw an error so the user knows what happened. Otherwise they just get an empty list with not indication that something went wrong.

Reply:
Changed the `return` into a `throw` statement, even if the GitHub PR UX refuses to display that...